src/lib/io/include/sol-memmap-storage.h

```diff
@@ -269,6 +342,16 @@ sol_memmap_write_bool(const char *name, bool value,
     return r;
 }
 
+/**
+ * @brief Reads an int32_t contents.
```

Comment:
I prefer to avoid writing a type here. I would write:
Reads a 32 bit integer content.